web/src/scripts/components/menu/FilterableList.jsx

```diff
@@ -138,6 +138,11 @@ export default class FilterableList extends Component {
 
     return _.orderBy(filteredList, [
       (pkg) => {
+        if (pkg.name) {
+          return pkg.name.split('/').length
```

Comment:
My understanding is that every time you do a split it compiles a RegEx anyway, so you might as well do it beforehand if it's a common operation. Believe it just saves on a constant factor?